Runtime/Core/BucketManagement/BucketStore.cs

```diff
@@ -129,39 +128,35 @@ public CreateBucketResponse CreateBucket(CreateBucketRequest request)
                         ErrorMessage = $"HTTP Status Code {putLoggingBucketResponse.HttpStatusCode}"
                     });
                 }
-
-                // Getting Existing ACL of the logging bucket
-                GetACLResponse getACLResponse = _amazonS3Wrapper.GetACL(new GetACLRequest
-                {
-                    BucketName = loggingBucketName
-                });
-
-                if (getACLResponse.HttpStatusCode != HttpStatusCode.OK)
-                {
-                    return Response.Fail(new CreateBucketResponse()
-                    {
-                        ErrorCode = ErrorCode.AwsError,
-                        ErrorMessage = $"HTTP Status Code {getACLResponse.HttpStatusCode}"
-                    });
-                }
-
-                S3AccessControlList s3AccessControlList = getACLResponse.AccessControlList;
-                s3AccessControlList.AddGrant(new S3Grantee { URI = "http://acs.amazonaws.com/groups/s3/LogDelivery" }, S3Permission.WRITE);
-                s3AccessControlList.AddGrant(new S3Grantee { URI = "http://acs.amazonaws.com/groups/s3/LogDelivery" }, S3Permission.READ_ACP);
-
-                // Grant logging access to the logging bucket
-                PutACLResponse putACLResponse = _amazonS3Wrapper.PutACL(new PutACLRequest
+                
+                var logPrefix = "GameLiftBootstrap";
+                var resourceArn = @"""arn:aws:s3:::" + loggingBucketName + "/" + logPrefix + @"*""";
+                var policy =  @"{
+                                ""Statement"":[{
+                                ""Sid"": ""S3ServerAccessLogsPolicy"",
+                                ""Effect"": ""Allow"",
+                                ""Principal"": { ""Service"": ""logging.s3.amazonaws.com"" },
+                                ""Action"": [""s3:PutObject""],
+                                ""Resource"": [" + resourceArn + @"],
+                                ""Condition"": {
+                                ""ArnLike"": { ""aws:SourceArn"": ""arn:aws:s3:::" + request.BucketName + @""" },
+                                ""StringEquals"": { ""aws:SourceAccount"": """ + request.AccountId + @""" }
+                                        }
+                                    }]
+                                }";
```

Comment:
I'm wondering if there's a better way to make this more readable, and make updates easier. Can this be stored as a json, and you apply String.format on the retrieved string to substitute the required variables?